src/Utilities/_DataValidation.php

```diff
@@ -65,6 +65,29 @@ function validateValueName(string $valueName): void
     }
 }
 
+if (!function_exists('validateItems')) {
+    function validateItems(array $items): void
+    {
+        foreach ($items as $item) {
```

Comment:
Can you also add a check to make sure the array itself isn't empty, please?

Reply:
I think it should be covered in this check?
If `$array = [""]`, this should throw `InvalidArgumentError`.
And that is checked in [this test](https://github.com/momentohq/client-sdk-php/pull/29/files#diff-0cf06050f3f70b72b13887aec2aaca681da458ce829b4335c999fcab3fd4c369R1118).
But please let me know if misunderstand your request!

Reply:
Oh I got it! `$array = []` 👍 
Please ignore my comment above.